pkg/crc/machine/types.go

```diff
@@ -19,7 +19,7 @@ type StartConfig struct {
 	NameServer string
 
 	// User Pull secret
-	PullSecret *cluster.PullSecret
+	PullSecret cluster.PullSecretLoader
```

Comment:
Is this actually still 'cluster' logic? It concerns the cluster ...
*just wondering*